multi_partner_learning.py

```diff
@@ -171,18 +170,18 @@ def compute_test_score(self):
                 model_to_evaluate = sequentially_trained_model
             elif self.learning_approach in ['fedavg', 'seq-with-final-agg', 'seqavg']:
                 model_to_evaluate = self.build_model_from_weights(self.aggregate_model_weights())
-            model_evaluation = model_to_evaluate.evaluate(
-                x_val, y_val, batch_size=constants.DEFAULT_BATCH_SIZE, verbose=0,
-            )
 
-            current_val_loss = model_evaluation[0]
-            current_val_metric = model_evaluation[1]
+
+            val_evaluation = self.evaluate_model(model_to_evaluate, self.val_data)
+
+            current_val_loss = val_evaluation[0]
+            current_val_metric = val_evaluation[1]
 
             self.score_matrix_collective_models[epoch_index, minibatch_count] = current_val_metric
             self.loss_collective_models.append(current_val_loss)
 
             logger.info(f"   Model evaluation at the end of the epoch: "
-                        f"{['%.3f' % elem for elem in model_evaluation]}")
+                        f"{['%.3f' % elem for elem in val_evaluation]}")
```

Comment:
Same here, `model_evaluation` is clearer. But I get the iead, you wanted to identify clearly evaluations on val data vs. test data. So let's use explicit names then: `model_eval_val_data` and `model_eval_test_data`.

Reply:
I will use this notation